main.go

```diff
@@ -333,7 +336,7 @@ blockingLoop:
 	return 0
 }
 
-func setupControllers(mgr ctrl.Manager, sw *watch.ControllerSwitch, tracker *readiness.Tracker, setupFinished chan struct{}) error {
+func setupControllers(mgr ctrl.Manager, sw *watch.ControllerSwitch, tracker *readiness.Tracker, setupFinished chan struct{}, ctx context.Context) error {
```

Comment:
by convention, ctx is the first argument

Reply:
updated